src/frontend/apps/impress/src/features/docs/doc-editor/utilsAI.ts

```diff
@@ -0,0 +1,145 @@
+import { Block as CoreBlock } from '@blocknote/core';
+
+type Block = Omit<CoreBlock, 'type'> & {
```

Comment:
why do we need this? I think you could use `Block` from BlockNote directly?

Reply:
I get some overlapping warning because of the unions, changing the type to string help to pass over. 
Not sure it is the best though.

![image](https://github.com/user-attachments/assets/8d513a84-802a-4c75-be4d-ddcfadc79e4f)

Reply:
got it. It's possible for sure but would require a bit more work on the typings. If it's a priority we can pair on this